include/pybind11/stl.h

```diff
@@ -305,6 +316,60 @@ template<> struct type_caster<std::experimental::nullopt_t>
     : public void_caster<std::experimental::nullopt_t> {};
 #endif
 
+#if defined(PYBIND11_HAS_FILESYSTEM)
+template<typename T> struct path_caster {
+
+    static PyObject* unicode_from_fs_native(const std::string& w) {
```

Comment:
These could still become `private`, but I don't per se need this, if there are no other commits being done.

Reply:
private it is

Reply:
Thanks! And apologies for all the nitpicking ;-)